docs/update-locking.md

```diff
@@ -35,7 +59,16 @@ Using the above-mentioned library, the lock can be acquired like in this CoffeeS
 			throw err if err?
 ```
 
-There are other libraries you can use in different languages, for example [this Python library](http://pythonhosted.org/lockfile/lockfile.html#examples).
+#### Python
+
+In Python you can use the for example the [`lockfile` library](http://pythonhosted.org/lockfile/lockfile.html#examples)
+```python
+from lockfile import LockFile
+lock = LockFile("/tmp/resin/resin-updates.lock")
+with lock:
+    print lock.path, 'is locked.'
+```
+Check the link for more examples, and other Python libraries that provide locking.
```

Comment:
NIT you can remove the comma from this sentence

Reply:
Done